spec/blocksync/readme.md

```diff
@@ -0,0 +1,64 @@
+---
+order: 
+parent:
+    title: blocksync
+    order: 
+---
+
+
+In a proof of work blockchain, syncing with the chain is the same process as staying up-to-date with the consensus: download blocks, and look for the one with the most total work. In proof-of-stake, the consensus process is more complex, as it involves rounds of communication between the nodes to determine what block should be committed next. Using this process to sync up with the blockchain from scratch can take a very long time. It's much faster to just download blocks and check the merkle tree of validators than to run the real-time consensus gossip protocol.
+
+The Blocksync Reactor's high level responsibility is to enable peers who are
+far behind the current state of the consensus to quickly catch up by downloading
+many blocks (that have already been decided) in parallel, verifying their commits, and executing them against the
+ABCI application.
+
+Tendermint full nodes run the Blocksync Reactor as a service to provide blocks
+to new or recovering nodes. The nodes run the Blocksync Reactor in "fast_sync" mode,
+where they actively make requests for more blocks until they sync up.
+Once caught up, "fast_sync" mode is disabled and the node switches to
+using the Consensus Reactor. 
+
+*Note* It is currently assumed that the Consensus reactor is already running. It is therefore not turned on by the Blocksync reactor. In case it has not been started, the Blocksync reactor simply returns.
+
+### Conditions to start Blocksync
+
+A node can switch to blocksync directly on start-up or after completing `state-sync`. Currently, switching back to blocksync from consensus is not possible. It is expected to be handled in [Issue #129](https://github.com/tendermint/tendermint/issues/129).
```

Comment:
Why "directly"? Please add a link to State Sync specification, and maybe briefly summarize what it does (this is useful for understanding other parts of this spec too).

Reply:
The second sentence, and the referred issue, are not really related to starting block sync, but to "restarting" it under certain circumstances. I would move it to another paragraph, in particular because it is hypothetical and not implemented yet.